main.go

```diff
@@ -84,6 +86,14 @@ func init() {
 	utilruntime.Must(obv1.AddToScheme(scheme))
 	utilruntime.Must(routev1.AddToScheme(scheme))
 	//+kubebuilder:scaffold:scheme
+
+	opts := zap.Options{}
+	opts.BindFlags(flag.CommandLine)
```

Comment:
when invoked from a container, we don't pass any command-line arguments.
in k8s, we should allow the user to set verbosity via configmap of the operator.
hence by changing the value in the config map and restarting the operator, we should allow the user to change verbosity level.
IMO we should prefer env-vars vs cli args, so we'll have the same behavior for the app when it runs locally via `make run` or when it runs in a pod.
Have a look at `OBC_AUTO_CREATE`

Reply:
Yes, it makes sense. I'll change it. Thanks